### ISSUE TITLE
Documentation: What is the format?

### ISSUE THREAD

**Issue by Meai** [sourcegraph/srclib#163]
At the end of the day srclib is a format right? Yet from the website and readme, the actual format is nowhere to be seen.
It's like making json but not actually showing what json looks like.

**Comment by MaikuMori** [comment]
The format can still change, [here's some information](https://srclib.org/plugins/creatingaplugin/#analyze-a-project).

Another option is to look at the structs in the source. Here are some:

 - [SourceUnit](https://github.com/sourcegraph/srclib/blob/master/unit/source_unit.go#L31)
 - [Def](https://github.com/sourcegraph/srclib/blob/master/graph/def.pb.go#L84)
 - [Ref](https://github.com/sourcegraph/srclib/blob/master/graph/ref.pb.go#L27)
 - ... and so on


Srclib team is currently working on autogenerated documentation from source, but the work is not yet done as far as I know. The source itself is quite well documented.

**Comment by sqs** [comment]
I agree: the format should be easy to get to from the homepage. Right now you have to click on Docs, then Schema, then one of the items.

**Comment by jelmerdereus** [comment]
@Meai thanks for your feedback. It's on the wish list already and it will be addressed.
